open-company/open-company-web
# Settings back

### Review after #850 

Card: https://trello.com/c/b11GsX4c

To test:
- open menu
- open My Profile
- [x] click Back, did it work?
- click on notifications
- click the X
- [x] back to menu?
- click on recurring updates
- click New
- add a new update
- save
- [x] back to the list of reminders?
- click on the reminder
- click back (or the X)
- [x] back to reminders?
- click back
- [x] back to menu?
- click org settings
- click Back
- [x] back to menu?
- click on invite
- click on X
- [x] back to menu
- click on Integrations
- click Back
- [x] back to menu?
- click on + to add a section
- click back or the X
- [x] back to the stream of posts (menu closed)?
- go to a section
- click on the section settings
- click Back or X
- [x] back to the stream of posts (menu closed)?
- now click on the WRT count of a post
- [x] WRT panel open?
- close it
- open WRT panel again
- from console have a notification: `oc.web.actions.notifications.show_notification(cljs.core.merge(oc.web.lib.utils.app_update_error, clojure.walk.keywordize_keys(cljs.core.js__GT_clj({"id": "test"}))))` (this will show the app update error with the bg color needed by the currently shown panels)
- [x] do you see a light theme notification?

This PR inherited a bug from #850 : the menu doesn't lock the page scroll so you can scroll the page behind the menu panel.

Test inherited issue:
- [x] can you scroll up and down when in All Posts?
- [x] can you scroll up and down when on a section?
- open the menu
- [x] can you NOT scroll the page behind the menu?
- [x] open all the panel accessible from the menu and check the page is still not scrolling
- [x] make sure the panels can scroll if they need: try with a particularly long panel and resize your window
- now open section settings
- [x] can you NOT scroll the page behind?
- now click on the + to add a new section
- [x] can you NOT scroll the page behind?
